src/mail/view/MailView.ts

```diff
@@ -545,20 +575,15 @@ export class MailView extends BaseTopLevelView implements TopLevelView<MailViewA
 	}
 
 	private moveMails() {
-		const mailList = this.cache.mailList
+		const mailList = this.mailViewModel.listModel
 		if (mailList == null) {
 			return
 		}
 
-		const selectedMails = mailList.list.getSelectedEntities()
+		const selectedMails = mailList.getSelectedAsArray()
 
 		// Render the dropdown at the position of the selected mails in the MailList
-		showMoveMailsDropdown(locator.mailModel, mailList.list.getSelectionBounds(), selectedMails)
-	}
-
-	private async switchToFolder(folderType: Omit<MailFolderType, MailFolderType.CUSTOM>): Promise<void> {
-		const mailboxDetail = assertNotNull(await this.getMailboxDetails())
-		m.route.set(this.folderToUrl[getElementId(assertSystemFolderOfType(mailboxDetail.folders, folderType))])
+		showMoveMailsDropdown(locator.mailModel, getMoveMailBounds(), selectedMails)
```

Comment:
i would just put it at the top of the mail column, if you select mail and then scroll them out of view before pressing v you get some funky results.

Reply:
didn't I already hardcode it?

Reply:
looks like it, but I wonder what causes this then:

[Screencast from 19.06.2023 10:51:40.webm](https://github.com/tutao/tutanota/assets/29049728/b764d048-0772-4c56-b296-403d5743f133)

(it seems to be generally messed up and not related to the scroll position)